<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CDialogDubbing</class>
 <widget class="QDialog" name="CDialogDubbing">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>494</width>
    <height>243</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>打开配音文件 </string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget
{
	font-size:15px;
	font-family:Microsoft Yahei;	
}</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>461</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="1">
     <widget class="QPushButton" name="BtnAudioFileOpen">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QComboBox" name="CmbAudioFile">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="BtnVideoFileOpen">
      <property name="maximumSize">
       <size>
        <width>90</width>
        <height>30</height>
       </size>
      </property>
      <property name="text">
       <string>浏览</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QComboBox" name="CmbVideoFile">
      <property name="minimumSize">
       <size>
        <width>0</width>
        <height>30</height>
       </size>
      </property>
      <property name="editable">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>配音:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>打开视频:</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>274</x>
     <y>200</y>
     <width>195</width>
     <height>35</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QPushButton" name="BtnOK">
      <property name="text">
       <string>确定</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="BtnCancel">
      <property name="text">
       <string>取消</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
